test/cmd/command.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"sync"
+
+	"knative.dev/pkg/test/helpers"
+)
+
+// Run will run the command and return the standard output, plus error if there is one.
+func Run(cmdLine string) (string, error) {
+	cmdSplit := strings.Fields(cmdLine)
+	if len(cmdSplit) == 0 {
+		return "", fmt.Errorf("the command line %q cannot be empty", cmdLine)
+	}
+
+	cmd := cmdSplit[0]
+	args := cmdSplit[1:]
+	cmdOut, err := exec.Command(cmd, args...).Output()
```

Comment:
You're swallowing stderr (actually, letting it flow to the terminal), you shouldn't.

Reply:
Nice catch. Added the stderr.